### ISSUE TITLE
Add Wire.setModule() to the Wire Library (I²C) reference page

### ISSUE THREAD

**Issue by rei-vilo** [energia/Energia#934]
Add `Wire.setModule()` to the [Wire Library (I²C) reference page](http://energia.nu/reference/wire/).

**Comment by duyka0110** [comment]
excuse me, what is the parameter? is it the address of the I2C pins?

**Comment by rei-vilo** [comment]
No, it is the port number. Please refer to the [pins map](http://energia.nu/wordpress/wp-content/uploads/2014/01/LaunchPads-LM4F-TM4C-—-Pins-Maps-11-32.jpeg).

**Comment by duyka0110** [comment]
for example the I2C0. i need 2 setModule, one for the SCL and one for the SDA, right?

**Comment by rei-vilo** [comment]
No, you need only one `setModule` for I²C port 0. You're selecting the I²C port, not the pins. The I²C port knows which pins to use.

**Comment by duyka0110** [comment]
okay thank you very much

**Comment by hemangjoshi37a** [comment]
I am using msp430 launchpad  and i want to change spi module from UCB0 to UCA0. What should i do???

**Comment by hemangjoshi37a** [comment]
In case If i want to change i2c communication then which pins will be availabe for i2c for wire.setModule(1) or wire.setModule(2) ?? means which pin is scl and sda for different modules.??

**Comment by rei-vilo** [comment]
Feel free to 

* Have a look at the data sheet of the exact MSP430 you're using, and 
* Switch to Code Composer Studio which provides full access to all features of the MSP430 you're using.

**Comment by StefanSch** [comment]
the pinmaps on energia,nu also shows this information as implemented in the Energia lib.
e.g.: http://energia.nu/pin-maps/guide_msp430fr5969launchpad/
shows behind SCL (x) where x is the id used for setModule

**Comment by StefanSch** [comment]
To change SPI from UCB0 to UCA0 you have to hack the energia source files, start with this files:
.\hardware\energia\msp430\variants\MSP-EXP430G2553LP\pins_energia.h
.\hardware\energia\msp430\libraries\SPI\utility\eusci_spi.cpp

**Comment by hemangjoshi37a** [comment]
@rei-vilo , i am not  using code composer studio.
@duyka0110 , Okay thax bro...

**Comment by robertinant** [comment]
@hemangjoshi37a please use the Energia forum @ http://forum.43oh.com/forum/119-energia/ for questions. Github issues as named are only for issues and not for questions. If it is determined that after you post on 43oh, your question turns out to have to do with a bug, then you can post the bug here on github.

**Comment by hemangjoshi37a** [comment]
@robertinant  
Okay bro sure...

**Comment by robertinant** [comment]
I have reopened it since this is still a valid issue. Wire.setModule() is currently not documented on the Energia reference page. You used this thread to ask a question hence I requested you to post it in the forum instead.
